cmd/kube-proxy/app/server.go

```diff
@@ -631,10 +631,10 @@ func (s *ProxyServer) birthCry() {
 // used for NodePort handling unless NodePortAddresses is not set to 'primary'.)
 //
 // The order of precedence is:
-//  1. if bindAddress is not 0.0.0.0 or ::, then it is used as the primary IP.
+//  1. if nodeIPOverride[0] is not 0.0.0.0 or ::, then ip family of nodeIPOverride[0] is considered as primary IP family.
 //  2. if rawNodeIPs is not empty, then its address(es) is/are used
 //  3. otherwise the node IPs are 127.0.0.1 and ::1
```

Comment:
This is incorrect at this point. (In particular, it never says that it actually uses the IPs in nodeIPOverride as the node IPs.)